StepDefinitions/TestSteps.cs

```diff
@@ -0,0 +1,131 @@
+using System;
+using TechTalk.SpecFlow;
+
+namespace TestSpecs {
+    [Binding]
+    class TestStpes
+    {
+        [Given(@"I am on the (.*) website")]
+        public void GivenIAmOnTheNLWebsite(string countryCode)
+        {
+            setCountryCode(countryCode);
+            Driver.Navigate().GoToUrl(baseURLCK);
+        }
+
+        [Given(@"All cookies are set")]
```

Comment:
Combine this step with `Given(@"I am on the (.*) website")`, as all your test only have 2 possibilities:
* User on a page with cookies
* User on a page without cookies
You could combine it like `Given(@"I am on the (.*) home page( with user cookies)?")`